regnety/dataset/imagenet.py

```diff
@@ -127,14 +134,37 @@ def _read_tfrecs(self) -> Type[tf.data.Dataset]:
             num_parallel_calls = AUTO 
         )
 
-        ds = ds.cache("tf_cache")
-       
-        ds = ds.repeat()
-        ds = ds.batch(self.batch_size)
+        ds = ds.cache("gs://adityakane-train/cache/"+str(time.time()))
```

Comment:
Let's get the caching address from config and use `os.path.join()`?

Reply:
Actually, this doesn't work as expected, since `os.path.join` uses "/" as a separator. It created a chain of directories inside the bucket. (gs: `/` ` / ` adityakane-train `/`... )

Reply:
I meant something like `os.path.join("gs://adityakane-train/cache", ...)`.